catenets/experiment_utils/metrics.py

```diff
@@ -0,0 +1,28 @@
+# third party
```

Comment:
should this file be called torch_metrics or so? there are definitely also (jax) metrics in experiment_utils/base